joomla/joomla-cms
# [staging] Deleting unused Contacts "banned" fields

Pull Request for Issue https://github.com/joomla/joomla-cms/pull/23569

### Summary of Changes
Deleting unused contact fields:
banned_email
banned_subject
banned_text

Keeping them in the Component Options as, since 2.5, it is the only place where they work.

Deprecating some ini strings
Corrected sql files
Modified some unit test files EXCEPT `/tests/unit/stubs/database/jos_finder_links.csv` which I have no idea how to clean.
For that one, @mbabker @wilsonge @Hackwar please make PR against mine 


### Testing Instructions
Edit a contact, a contact category, all contacts menu items and make sure the 3 fields do not appear.
Check the sql for possible errors.
